media-libs/libnspsl/libnspsl-0.1.5.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="decoding library for BMP and ICO image file formats, written in C"
+HOMEPAGE="http://www.netsurf-browser.org/projects/"
+SRC_URI="https://download.netsurf-browser.org/libs/releases/${P}-src.tar.gz"
+
+LICENSE="MIT"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~m68k-mint"
+IUSE=""
+
+DEPEND="
+	dev-util/netsurf-buildsystem
```

Comment:
Looking at the content of this package, I think this should go in DEPEND and BDEPEND.

Reply:
Sure? Why both?